.github/actions/integration-tests/ci-environment-info

```diff
@@ -2,10 +2,60 @@
 # -*- mode: sh; indent-tabs-mode: nil; sh-basic-offset: 4 -*-
 # vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=bash
 
+SCRIPT_DIR=$(dirname $0)
+SCRIPT_DIR=$(readlink -e ${SCRIPT_DIR})
+
+. ${SCRIPT_DIR}/base
+
+CI_RUN_ENVIRONMENT="standalone"
```

Comment:
ok so default is local env

Reply:
Correct.  Since I was able to hard code using github mode into the action it made sense to make standalone mode the default whenever somebody runs it somewhere else (which by definition is almost certainly not github).